frontend/awx/resources/inventories/inventoryHostsPage/inventoryHostsPage.tsx

```diff
@@ -75,7 +75,10 @@ export function InventoryHostsPage() {
           page: AwxRoute.InventoryHosts,
```

Comment:
Change name of file because this view shows the groups of a single host.
```InventoryHostPage.tsx```

Reply:
done